ChessBoard/ChessBoardDialog.cs

```diff
@@ -26,5 +26,25 @@ public void GameStart()
             }
             Console.WriteLine("You entered: " + _inputNumberOfRows);
         }
+
+        public bool TryAgain()
+        {
+            Console.WriteLine();
+            Console.WriteLine("Try code again? \n1 - yes \nany key - close application");
```

Comment:
Inline formatting is not considered to be a good practice, I'd suggest using either separate writelines or stringBuilder